lib/freefem-runner.js

```diff
@@ -33,7 +33,7 @@ export default {
   config: {
     path: {
       'type': 'string',
-      'description': 'Path to the FreemFem++ executable',
+      'description': 'Path to the FreeFEM executable',
       'default': 'FreeFem++'
```

Comment:
Is the executable name has changed too ?
Because this variable is used by default to run the code.

Reply:
The executable name is still `FreeFem++`